index.html

```diff
@@ -14,18 +14,31 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <header class="header">
-        <a href="#" class="logo">A.Ahad</a>
+    <header>
+        <div class="header">
+        <div class="logo">
+        <a href="#">A.Ahad</a>
+        </div>
         <nav >
+            <ul>
+                <li><a href="#">Portfolio</a>
+                </li>
+                <li><a href="#">About</a></li>
+                <li>
+                    <a href="#">Contact</a>
+                </li>
```

Comment:
[ Optional ] Please consider making these links scroll to the respective sections of the page, `Portfolio` should scroll to the portfolio section `About` should scroll to the about section, `Contact` should scroll to the contact section. You can achieve this using the code below.
```HTML
<ul>
   <li><a href="#projects">Portfolio</a> </li>
     <li><a href="#about">About</a></li>
       <li><a href="#contact">Contact</a></li>
</ul>

<section id="portfolio"></section>
<section id="about"></section>
<section id="contact"></section>
```